portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,10 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
```

Comment:
FYI some tools (including the GitHub review tool, apparently) will complain at you if your files don't end with a newline. I've never seen anything bad happen from that though.